src/engine/Server.h

```diff
@@ -95,7 +95,8 @@ class Server {
   std::weak_ptr<ad_utility::websocket::QueryHub> queryHub_;
 
   boost::asio::static_thread_pool queryThreadPool_;
-  boost::asio::static_thread_pool updateThreadPool_{1};
+  static constexpr size_t UPDATE_THREAD_POOL_SIZE = 1;
+  boost::asio::static_thread_pool updateThreadPool_{UPDATE_THREAD_POOL_SIZE};
```

Comment:
See my other comment (still leave this constant and add a comment here that this has to be  1 because of the atomicity of the updates.